synonymdev/bitkit-react-native
# fix: UI fixes

### Description

- make sure 'clipboard ease-of-use' redirect is only offered after AuthCheck
- fix list item touchable opacity
- Send: fix the loading state after auth (w/ PIN for payments enabled)
- Activity: pass `activityItem` directly to notification to make sure there is no rerender or empty notification
- chore: add env var to ignrore LDK logs

### Type of change


### Tests


### QA Notes

1. enable clipboard ease-of-use

- app start / normal -> no redirect
- app start / with deep link -> should redirect
- app start / with clipboard data -> should offer redirect
- app to foreground / normal -> no redirect
- app to foreground / with deep link -> should redirect
- app to foreground / with clipboard data -> should offer redirect

2. enable PIN on launch & PIN when idle

- app start w/ pinOnLaunch / normal -> should show PIN check
- app start w/ pinOnLaunch / with deep link -> should show PIN check and redirect
- app start w/ pinOnLaunch / with clipboard data -> should show PIN check and offer redirect
- app to foreground w/ pinOnIdle / normal -> should show PIN check
- app to foreground w/ pinOnIdle / with deep link -> should show PIN check and redirect
- app to foreground w/ pinOnIdle / with clipboard data -> should show PIN check and offer redirect
